src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -273,14 +273,13 @@ describe('ListOfPublishedProfiles', () => {
 
     // Only this button isn't disabled, because it's the only one with the JWT information.
     const workingButton = getDeleteButtonForProfile('#012345');
-    expect(workingButton.disabled).toBe(false);
+    expect(workingButton).toBeEnabled();
 
     // All others are disabled.
     const allButtons = getAllByText('Delete');
     for (const button of allButtons) {
       if (button !== workingButton) {
-        // $FlowExpectError findAllByText returns HTMLElement, but we know these are buttons.
-        expect(button.disabled).toBe(true); // eslint-disable-line jest/no-conditional-expect
+        expect(button).toBeDisabled(); // eslint-disable-line jest/no-conditional-expect
```

Comment:
Note: it's not really flow-checked anymore, because we can put anything in `expect`?